File: scripts/inspect_file.py
import re

with open('src/components/ArticleRenderer.tsx', encoding='utf-8') as f:
    content = f.read()

# Find the exact lines 927-938
lines = content.split('\n')
print(f"Total lines: {len(lines)}")
for i in range(925, 940):
    print(f"Line {i+1}: {repr(lines[i])}")

File: scripts/inspect_file.js
const fs = require('fs');
const content = fs.readFileSync('src/components/ArticleRenderer.tsx', 'utf-8');
const lines = content.split('\n');
console.log(`Total lines: ${lines.length}`);
for (let i = 925; i < 940; i++) {
    console.log(`Line ${i+1}: ${JSON.stringify(lines[i])}`);
}
